core/modules/date/templates/date-views-pager.tpl.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @file
+ * Template file for the example display.
+ *
+ * Variables available:
```

Comment:
Can we format the `Available variables:` section  to match the rest of core's templates, for example:

```php
 * Available variables:
 * - $plugin: The pager plugin object. This contains the view.
 * - $plugin->view: The view object for this navigation.
 * - $nav_title: The formatted title for this view. In the case of block
 *   views, it will be a link to the full view, otherwise it will
 *   be the formatted name of the year, month, day, or week.
 * - $prev_url: Url for the previous calendar page. The link is created
 *    in the template to make it easier to change the text, add images, etc.
 * - $next_url: Url for the next calendar page. The link is created
 *    in the template to make it easier to change the text, add images, etc.
```